### ISSUE TITLE
Avoid inline styles with !important

### ISSUE THREAD

**Issue by alvaromontoro** [jackdomleo7/Checka11y.css#50]
### Describe the new a11y feature or project enhancement

Users may define their own stylesheet for accessibility purposes. Inline styles can only be overridden using `!important`, and inline styles with `!important` cannot be overridden via a user stylesheet, so they should be avoided.

From [WebAIM](https://webaim.org/techniques/css/): "_To improve accessibility, end users may change browser default styles and may define custom styles that override any other styles in the "cascade". Pages must be adaptable and flexible to end user style customizations._"

### Describe the solution you'd like

Show an error/warning if inline styles are present and they have an `!important`, as users won't be able to override those styles with a custom stylesheet. The CSS rule could be like this:

```css
[style*="!important"] {
  /* error/warning */
}
```

### Link(s)

- [WebAIM: Accessible CSS](https://webaim.org/techniques/css/#intro)
- [MDN attribute selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors)
- [MDN: The important exception](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#The_!important_exception)

**Comment by jackdomleo7** [comment]
This sounds like a fantastic feature!!!

**Comment by alvaromontoro** [comment]
Can I take this one? I know I have a different one assigned, but it is "blocked" by the SCSS change
